src/main/kotlin/no/nav/syfo/unntak/api/UnntakApi.kt

```diff
@@ -55,8 +57,26 @@ fun Route.registerUnntakApi(
                 call.respond(unntakDTOList)
             }
         }
-        get(unntakApiHackaton) {
-            call.respond(unntakService.getHackaton(call.getNAVIdent()))
+        get(unntakApiStatistikk) {
+            val unntakForventetFriskmelding = unntakService.getUnntakForVeileder(call.getNAVIdent()).filter { unntak ->
```

Comment:
Bør vi også sjekke at veileder har tilgang til personen unntaket gjelder her? Ser vi gjør det ved henting av dialogmøter for veileder.

Reply:
Vi returnerer jo ingenting om innbyggerne, så jeg tenker det er fint at vi får alle i statistikken, selv om veilederen evt. ikke lenger har tilgang.